server.js

```diff
@@ -443,4 +441,8 @@ app.delete('/api/v1/sounds/:id', (request, response) => {
   })
 })
 
+app.get('*', function (request, response) {
```

Comment:
might want to edit the `*` route match eventually, but if it work down here that's cool for now